sphericart/CMakeLists.txt

```diff
@@ -13,7 +13,7 @@ OPTION(BUILD_SHARED_LIBS "Build shared libraries instead of static ones" OFF)
 OPTION(SPHERICART_BUILD_TESTS "Build and run tests for Sphericart" OFF)
 OPTION(SPHERICART_OPENMP "Try to use OpenMP when compiling Sphericart" ON)
 OPTION(SPHERICART_ARCH_NATIVE "Try to use -march=native when compiling Sphericart" ON)
-option(SPHERICART_ENABLE_CUDA "Are we building the CUDA backend of Sphericart?" ON)
+OPTION(SPHERICART_ENABLE_CUDA "Are we building the CUDA backend of Sphericart?" ON)
```

Comment:
So I don't love that this option (and `SPHERICART_OPENMP`) are silently ignored if we can not find the right compiler / compiler flag. It was already the case before, so I'll open a separate issue to discuss it more.